index.js

```diff
@@ -538,6 +562,19 @@ class DHT extends EventEmitter {
           req.sendReply(0, null, false, false)
           return
         }
+        case DELAYED_PING: {
+          if (req.value === null || req.value.byteLength < 4) return
+          const delayMs = c.uint32.decode({ start: 0, end: 4, buffer: req.value })
+          // cap the delay at 10 seconds
```

Comment:
This comment will be outdated if we ever change `MAX_DELAY_MS`. Better to not have the comment, or just say 'cap the delay' (in this case `if (delayMs > MAX_DELAY_MS) return` is sufficiently self-documenting that it doesn't need the comment IMO)